src/client/pages/OfferNew/Compare/CompareTable.tsx

```diff
@@ -288,7 +288,7 @@ const MobileDropdownWrapper = styled('div')`
   max-width: 85px;
   margin-bottom: 3.25rem;
 `
-const TinyDownArrow = styled(DownArrow)`
+const TinyDownArrow = styled(ChevronDown)`
```

Comment:
Not crucial for this PR but this styled TinyDownArrow could be replaced then with `<ChevronDown size="7" />` :)